modules/Bio/EnsEMBL/ChainedAssemblyMapper.pm

```diff
@@ -276,7 +276,7 @@ sub map {
   throw('Incorrect number of arguments.') if(@_ < 6);
 
   my ($self, $frm_seq_region_name, $frm_start,
-      $frm_end, $frm_strand, $frm_cs, $fastmap, $to_slice) = @_;
+      $frm_end, $frm_strand, $frm_cs, $fastmap, $to_slice, $include_org_coord) = @_;
```

Comment:
Again, sorry, missing this new 8th argument flag description.

Reply:
You're perfectly right and I was aware of this inconsistency, thanks for the reminder! :+1: